requirements.txt

```diff
@@ -23,7 +23,7 @@ six==1.14.0
 sortedcontainers==2.1.0
 tables==3.6.1
 tdev2 @ git+https://github.com/bootphon/tdev2.git@a72467faf209a4f4ee90e81406a07dbfebe51e00
-torch==1.2.0
+torch==1.8.1
```

Comment:
Temporary fix for #116. Are we fine with that or shoud we solve the problem differently?